samples/Tutorials/CSharpBeginner/CSharpBeginner/CSharpBeginner.Game/Code/LoadingContentDemo.cs

```diff
@@ -13,21 +13,22 @@ namespace CSharpBeginner.Code
     public class LoadingContentDemo : SyncScript
     {
         private Model loadedMannequinModel = null;
-        private Stack<Entity> spawnedEntities = new Stack<Entity>();
```

Comment:
Are these constructor changes manual? There's little value to doing that.

Reply:
Oh, sorry. I've tried to not do any stylistic change this time, but this one has slipped in.
(the value btw is that I like to separate readonly fields fron non-readonly)